GANDLF/utils/modelio.py

```diff
@@ -39,6 +42,42 @@ def save_model(model_dict, path):
         model_dict["git_hash"] = None
     torch.save(model_dict, path)
 
+    try:
+        onnx_path = path.replace("pth.tar", "onnx")
+        dummy_input = torch.randn((1, num_channel, input_shape[0], input_shape[1]))
+
+        with torch.no_grad():
+            torch.onnx.export(
+                model,
+                dummy_input.to("cpu"),
+                onnx_path,
+                opset_version=11,
+                export_params=True,
+                verbose=True,
+                input_names=["input"],
+                output_names=["output"],
+            )
+
+        ov_output_dir = os.path.dirname(os.path.abspath(path))
+    except:
```

Comment:
```suggestion
    except RuntimeWarning:
```
To fix the codacy warning.